### DISCUSSION TITLE
Auto-open DevTools using preview option?

### DISCUSSION THREAD

**Discussion by wh81752** [Mogztter/asciidoctor-web-pdf#486] [Ideas]
When using option --preview is there a way to auto-open Chromium's DevTools?

Passing option `--auto-open-devtools-for-tabs` appears to do the job using Chrome Browser according to this article: https://developer.chrome.com/docs/devtools/open

So I wonder whether there is a way to pass this and perhaps other options down to Chromium.

**Comment by ggrossetie** [comment]
That's a good, I've created an issue to implement this feature.
